common/actions/src/com/google/idea/common/actions/ReplaceActionHelper.java

```diff
@@ -67,7 +68,7 @@ public static void replaceAction(
   }
 
   /** Wraps an action and makes it conditionally invisible. */
-  private static class RemovedAction extends AnAction {
+  private static class RemovedAction extends AnAction implements PossiblyDumbAware {
```

Comment:
The explicit implements here seems to be redundant.

Reply:
You're absolutely right. Removed.